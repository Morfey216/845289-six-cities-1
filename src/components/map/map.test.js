import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map';

const mock = [{
  title: `Offer Title`,
  type: `Offer Type`,
  image: `Image Path`,
  price: 0,
  rating: 100,
  coordinates: [52.3909553943508, 4.85309666406198],
}];

it(`Map correctly renders`, () => {
  const mapSection = document.createElement(`section`);
  mapSection.setAttribute(`id`, `mapid`);
  document.body.appendChild(mapSection);

  const tree = renderer
      .create(<Map offers={mock}/>)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
