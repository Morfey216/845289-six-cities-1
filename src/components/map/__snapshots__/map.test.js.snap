// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map correctly renders 1`] = `
<section
  className="cities__map map"
  id="mapid"
  style={
    Object {
      "height": 800,
      "width": "100%",
    }
  }
/>
`;
